.github/ISSUE_TEMPLATE/2_bug.yml

```diff
@@ -0,0 +1,136 @@
+name: Report a bug
+description: |
+  You found a bug in Flutter causing your application to crash or
+  throw an exception, a widget is buggy, or something looks wrong.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for using Flutter!
+
+        If you are looking for support, please check out our documentation
+        or consider asking a question on Stack Overflow:
+
+          - https://flutter.dev/
+          - https://api.flutter.dev/
+          - https://stackoverflow.com/questions/tagged/flutter?sort=frequent
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the [existing issues](https://github.com/flutter/flutter/issues)
+        required: true
+      - label: I have read the [guide to filing a bug](https://flutter.dev/docs/resources/bug-reports)
+        required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: Please tell us exactly how to reproduce the problem you are running into.
+      placeholder: |
+        1. ...
+        2. ...
+        3. ...
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected results
+      description: Please tell us what is expected to happen
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Actual results
+      description: Please tell us what is actually happening
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Code sample
+      description: |
+        Please create a minimal reproducible sample that shows the problem
+        and attach it below between the lines with the backticks.
+
+        To create it you can use `flutter create bug` command and update the `main.dart` file.
+
+        Alternatively, you can use https://dartpad.dev/
+        which is capable of creating and running small Flutter apps.
+
+        If your sample includes changes to native code, ideally you can create
+        a public Github repository and paste the link to it here
```

Comment:
missing full stop

Reply:
Actually, I would shorten the whole thing and list both alternatives in a single sentence, something like "Alternatively, you can use https://dartpad.dev/ or create a public GitHub repository to share your sample.”